common/wbdisable/wbdisable-en.html

```diff
@@ -0,0 +1,19 @@
+---
+{
+	"title": "Basic HTML",
+	"language": "en",
+	"altLangPage": "wb-disable-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-07-04"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<h2>Basic HTML mode</h2>
+<p>Click on the link below to enable Basic HTML Mode</p>
+<a href="?wbdisable=true">Switch to Basic HTML Mode</a>
```

Comment:
I suggest to add the 2 link that allow to go in basic HTML mode and the other link that go in the standard mode.

I recommend to have anchor `<a>` inside an explicit phrasing container element, like a paragraph `<p>`, instead on relying on the implicit paragraph.

Something like:
```suggestion
<p>Click on the link below to enable Basic HTML Mode</p>
<p><a href="?wbdisable=true">Switch to Basic HTML Mode</a></p>
<p><a href="?wbdisable=false">Switch to regular page</a></p>
```